main.py

```diff
@@ -126,10 +116,20 @@ async def submit_experiment(
         priority: Higher value means sooner scheduling.
         timed: The due date for the experiment in ISO format.
           None for no due date.
+        visualize: If True, the experiment file is modified for visualization.
+          The original file and vcd file are saved in the visualize path set in config.json.
     
     Returns:
         The run identifier, an integer which is incremented at each experiment submission.
     """
+    if visualize:
```

Comment:
Why does this `if visualize:` appear twice?

Can it be unified with the next `if visualize:` block? Are there any prerequisites before defining `expid`?

Reply:
Oh, it is a good question!

In the later PR, I will assign `expid` differently according to if `visualize` is `True` or `False`.
For this, I made `if visuzlie:` block before and after `remote.submit()`.

Reply:
Hmm, I do not know how complex it is, but if the blocks are short enough, I would rather use:
```
if visualize:
    A
    B
    C
else:
    B
```
instead of:
```
if visualize:
    A
B
if visualize:
    C
```
Is the `B` part so complex that it must be shared by two situations?

Reply:
I guess this can be handled in future PR. I will approve this merge.

Reply:
In this case, I am not sure that your suggestion works because there is an `else:` for the first `if:`.
In addition, since this project is not that performance-critical, we prefer readability to performance unless it is way too inefficient.
I think `if visualize:` is not expensive in such a sense!